roles/create-workspace/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: 'Create build dir for {{ image }}'
```

Comment:
What is the build dir used for? How does it play a role in creating an image from a Dockerfile?

Reply:
It is a local directory that serves as a build context - Ansible creates Dockerfile from template there and adds container-assets.